### PR TITLE
Update media cluster for spec 1.0

### PR DESCRIPTION
**Pull request by lazarkov**
#### Problem
- ChangeChannelResponse data structure is changed, changeChannel is replaced with data
- MediaPlaybackResponse is missing data parameter 

#### Change overview
see above

#### Testing
Run `./scripts/run_in_build_env.sh \  
                  "./scripts/tests/run_test_suite.py \
                     --chip-tool ./out/debug/standalone/chip-tool \
                     run \
                     --iterations 1 \
                     --all-clusters-app ./out/debug/standalone/chip-all-clusters-app \
                     --tv-app ./out/debug/standalone/chip-tv-app \
                  "`

### CODE REVIEW

File: examples/tv-app/linux/include/media-playback/MediaPlaybackManager.cpp [hunk 1]
```diff
@@ -141,6 +151,7 @@ void MediaPlaybackManager::HandleSeek(CommandResponseHelper<Commands::PlaybackRe
     if (positionMilliseconds > mDuration)
     {
         Commands::PlaybackResponse::Type response;
+        response.data   = chip::MakeOptional(CharSpan::fromCharString("data response"));
```
Comment (bzbarsky-apple) [inline examples/tv-app/linux/include/media-playback/MediaPlaybackManager.cpp:154]: These values don't really make sense.  Shouldn't we just be leaving `response.data` not set, if we have nothing useful to communicate?

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: examples/tv-app/linux/include/media-playback/MediaPlaybackManager.cpp (hunk 1 above)
Comment (lazarkov) [reply examples/tv-app/linux/include/media-playback/MediaPlaybackManager.cpp:154]: It's only for the testing purposes. As you know, the previous SDK was not supporting  char string as byte string was supported only. So to have test verification that this field is actually defined as string, that's why this random value is returned. 
The reasoning is that if null was used, there's no type verification.

Comment (lazarkov) [reply examples/tv-app/linux/include/media-playback/MediaPlaybackManager.cpp:154]: If that's wrong reasoning or there's a better way to achieve the verification, I'm all ears.

Comment (lazarkov) [reply examples/tv-app/linux/include/media-playback/MediaPlaybackManager.cpp:154]: @bzbarsky-apple

**Review by bzbarsky-apple — APPROVED** [review]
I guess if it's clear that this stuff is test-only and should not get shipped for real....

**Pull request merged**